frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTRoomArea.js

```
@@ -0,0 +1,150 @@
+      ? { code: `${formData?.usageCategoryMajor?.code}`, i18nKey: `PROPERTYTAX_BILLING_SLAB_OTHERS` }
+      : formData?.usageCategoryMajor
+  );
+  //   const { data: Menu, isLoading } = Digit.Hooks.pt.usePropertyMDMS(stateId, "PropertyTax", "OccupancyType");
+
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  const stateId = Digit.ULBService.getStateId();
+  const { data: Menu = {}, isLoading: menuLoading } = Digit.Hooks.pt.usePropertyMDMS(stateId, "PropertyTax", "UsageCategory") || {};
+  let usagecat = [];
+  usagecat = Menu?.PropertyTax?.UsageCategory || [];
+  let i;
+  let menu = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'menu' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)